nav/nav_master/nav_master.py

```
@@ -1,59 +1,74 @@
+            nav_models = ['*']
+        if nav_techniques is None:
+            nav_techniques = ['correlate_all']  # ['*']
 
         self._obs = obs
-        if nav_models is not None and not all(
-                x in ['stars', 'bodies', 'rings', 'titan'] for x in nav_models):
-            raise ValueError(f'Invalid nav_models: {nav_models}')
+        for nav_model in nav_models:
+            if any(fnmatch.fnmatch(x, nav_model) for x in valid_models):
+                continue
+            raise ValueError(f'Invalid nav_model: {nav_model}')
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)